test/es-module/test-esm-import-attributes-errors.js

```diff
@@ -55,6 +55,16 @@ async function test() {
     import(jsonModuleDataUrl, { with: { type: 'unsupported' } }),
     { code: 'ERR_IMPORT_ATTRIBUTE_UNSUPPORTED' }
   );
+
+  await rejects(
+    import(jsonModuleDataUrl, { assert: { type: 'json' } }),
+    { code: 'ERR_IMPORT_ATTRIBUTE_MISSING' }
+  );
+
+  await rejects(
+    import(`data:text/javascript,import${JSON.stringify(jsonModuleDataUrl)}assert{type:"json"}`),
+    SyntaxError
```

Comment:
This currently throws "unexpected token 'assert" -- I submitted a patch to V8 to make the error explicitly recommend to use 'with' instead: https://chromium-review.googlesource.com/c/v8/v8/+/5376260